src/editors/sharedComponents/TinyMceWidget/pluginConfig.js

```diff
@@ -14,6 +14,7 @@ const pluginConfig = ({ isLibrary, placeholder, editorType }) => {
   const quickToolbar = editorType === 'expandable' ? plugins.quickbars : '';
   const inline = editorType === 'expandable';
   const toolbar = editorType !== 'expandable';
+  const defaultFormat = editorType === 'text' ? 'p' : 'div';
```

Comment:
This causes errors for the the explanation field. The current code expects the explanation field to be wrapped in paragraph tags to properly save and add the explanation header. When the block uses `div` tags the explanation is not added.